scripts/dockerrun.sh

```diff
@@ -115,6 +115,7 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     -e GITHUB_PASSWORD \
     -e CLI_NUGET_FEED_URL \
     -e CLI_NUGET_API_KEY \
+    -e SIGNED_PACKAGES \
```

Comment:
I think this should be given the full name, `PUBLISH_SIGNED_PACKAGES` (or `PUBLISH_MANAGED_PACKAGES`) because I believe it still builds them. (But with `EnvVars.GetBool` this line shouldn't be needed.)